spec/features/user_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe 'User' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.